src/templates/en/2019/base_chapter.html

```diff
@@ -17,6 +17,36 @@
 indexBoxTitle.addEventListener('click', function(e) {
   indexBox.classList.toggle('show');
 });
+
+fetch(`https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}.json`)
+    .then(function(r) { return r.json(); })
+    .then(function(r) {
+  if (!r) {
+    return;
+  }
+
+  var comments = +r.posts_count - 1;
+  if (isNaN(comments)) {
+    return;
+  }
+  var el = document.getElementById('num_comments');
+  el.innerText = comments + ' ' + (comments == 1 ? 'comment' : 'comments');
+});
```

Comment:
This whole section has been moved to [static/js/chapter.js](https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/static/js/chapter.js) so should not be here any longer.